sphinx/ext/graphviz.py

```diff
@@ -197,6 +197,15 @@ def render_dot(self, code, options, format, prefix='graphviz'):
     return relfn, outfn
 
 
+def warn_for_deprecated_option(self, node):
+    if hasattr(self.builder, '_graphviz_warned_inline'):
+        return
+
+    if 'inline' in node:
+        self.builder.warn(':inline: option for graphviz is deprecated.')
```

Comment:
proposal: indicating 'since <version>' is good practice for deprecation warning.